src/components/pages/lessons/overlay/rate-course-overlay.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import {Formik} from 'formik'
+
+const rangeArr = [1, 2, 3, 4, 5, 6, 7]
+
+const rangeArrMobile = [
+  'Not at all likely',
+  'Very unlikely',
+  'Somewhat unlikely',
+  'Neutral',
+  'Somewhat likely',
+  'Very likely',
+  'Entirely likely',
+]
```

Comment:
this will be needed (probably, depends on Vojta's design farther recommendations) for the rate slider in mobile version